zemljevid.js

```diff
@@ -273,29 +283,56 @@ function izrisi_OJPP(odg) {
             bear = odg["direction"];
             speed = odg?.["vehicleSpeed"] ?? `<span style="font-size: xx-small;">${odg?.["operator_name"]}</span>` ?? "🤷🏻‍♀️";
             vid = odg["vehicle_id"];
+            timeDeparture = odg?.["time_departure"] ?? "";
+            timeArrival = odg?.["prihodNaCilj"] ?? "";
 
 
-            content=""
-            content +="<div class=''>"
-            content += `<a href="https://ojpp.si/trips/${odg?.["trip_id"]}" target="_blank" class="popup_route">${odg?.["route_name"]}</a>`; //LINIJA
-            content += ("<span class='popup_id' style='user-select: text'>Številka avtobusa: " + vid + "</span>"); //ID
+            vsebina = "";
+            vsebina += `
+                <div style="padding-left: 10px; padding-top: 10px;">
+                <span class="material-symbols-outlined" style="font-size: 2em; transform:translate(0,0.25em); z-index:100; color:var(--color-primary)" onclick="mymap.closePopup()">arrow_back</span>
+                <span class="popup-relacija">${odg?.["route_name"]}</span>
```

Comment:
Route name must be clickable (link to that trip on OJPP). Good job for everything else!